openslides_votecollector/static/js/openslides_votecollector/base.js

```diff
@@ -73,7 +73,7 @@ angular.module('OpenSlidesApp.openslides_votecollector', ['OpenSlidesApp.users']
                 }
             },
             relations: {
-                belongsTo: {
+                hasOne: {
```

Comment:
I would say it is inverse: Here belongsTo and hasOne at seat. See http://www.js-data.io/docs/examples, http://stackoverflow.com/questions/34565360/difference-between-hasone-and-belongsto-in-sequelize-orm or http://stackoverflow.com/questions/37582848/what-is-the-difference-between-belongsto-and-hasone-in-laravel.